subgraph.mainnet.yaml

```diff
@@ -26,9 +26,9 @@ dataSources:
     name: "Aggregator-0_2_0"
     network: "mainnet"
     source:
-      address: "0x0818Ad7016138f0A40DFAe30F64a923c2A8F61bA"
+      address: "0xC5519f3fcECC8EC85caaF8836563dEe9a00080f9"
```

Comment:
Why the change in address here?

Reply:
`0x0818Ad7016138f0A40DFAe30F64a923c2A8F61bA` corresponds to the v2 of the `ChainlinkConversionPath` contract, but for mainnet it was never used because the v2 only fixed an issue for other networks.

But if in the index.json file that list the contract's addresses we put the v2 address, it will be targetted instead of the v1 when doing aggregators updates.

That's why we v2 address was replaced by the v1, and I guess that Alex's update automatically updated this.

Reply:
Yes, this change was made automatically by `yarn prepare` after updating the `@requestnetwork/smart-contracts` package. The new contract version [0x0818Ad7016138f0A40DFAe30F64a923c2A8F61bA](https://etherscan.io/address/0x0818Ad7016138f0A40DFAe30F64a923c2A8F61bA) was deprecated, all aggregator updates are made on [0xC5519f3fcECC8EC85caaF8836563dEe9a00080f9](https://etherscan.io/address/0xC5519f3fcECC8EC85caaF8836563dEe9a00080f9), so both versions are the same contract.